dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcConnectionPool.java

```diff
@@ -64,18 +65,20 @@ public interface JdbcConnectionPool {
      * @param config configuration of connection pool
      * @return a new instance configured from the provided config
      */
+    @SuppressWarnings({"unchecked", "rawtypes"})
     static JdbcConnectionPool create(Config config) {
-        return JdbcConnectionPool.builder()
-                .config(config)
-                .build();
-    }
-
-    /**
-     * Create a fluent API builder for a JDBC Connection pool based on URL, username and password.
-     * @return a new builder
-     */
-    static Builder builder() {
-        return new Builder();
+        List<JdbcConnectionPoolProvider> poolProviders = HelidonServiceLoader
```

Comment:
no need to use builder here, just use `create` if not customizing it.

Reply:
fixed